lib/srv/authhandlers.go

```diff
@@ -404,12 +417,25 @@ func (h *AuthHandlers) UserKeyAuth(conn ssh.ConnMetadata, key ssh.PublicKey) (*s
 		return permissions, nil
 	}
 
+	ca, err := h.authorityForCert(types.UserCA, cert.SignatureKey)
```

Comment:
@capnspacehook Can you please add a comment that this check is needed even if we don't need the full RBAC to check to make it less likely that someone will remove it in the future?

Reply:
Good call, done